src/components/createField.jsx

```diff
@@ -96,8 +96,7 @@ export default function connectField(options) {
 
         /* Compose the field path */
         this.__fieldPath = fieldGroup ? [...fieldGroup, name] : [name]
-
-        invariant(
```

Comment:
Removed invariant as it will throw exception and halt the component rendering, however, error just logs the message in console and render null if Form is not passed

Reply:
Please could you also remove the invariant import at the beginning of the file?
It's not used here after your changes.